CELDEV-412

gcohen08 opened the pull request:
https://synjira.atlassian.net/browse/CELDEV-412

GalCohen08 committed:
New Package Celements-struct-component

GalCohen08 committed:
SelectTagPageTypeTest

GalCohen08 committed:
OptionTagPageType
OptionTagPageTypeTest

GalCohen08 committed:
OptionTagPageType
OptionTagPageTypeTest

GalCohen08 committed:
OptionTagPageTypeTest

GalCohen08 committed:
pom.xml

GalCohen08 committed:
deleting bin directory
web-module in parent directory

msladek commented on component/src/main/java/com/celements/structEditor/fields/OptionTagPageType.java:
```diff
@@ -26,7 +26,7 @@ public String getName() {
 
   @Override
-  protected String getViewTemplateName() {
+  public String getViewTemplateName() {
```
why changed to public?

msladek commented on component/src/main/java/com/celements/structEditor/fields/SelectTagPageType.java:
```diff
@@ -27,7 +27,7 @@ public String getName() {
 
   @Override
-  protected String getViewTemplateName() {
+  public String getViewTemplateName() {
```
why changed to public?

msladek commented on component/src/test/java/com/celements/structEditor/OptionTagPageTypeTest.java:
```diff
@@ -0,0 +1,185 @@
+
+  @Test
+  public void test_GetViewTemplateName() {
+    String expectedStr = "OptionTagView";
```
use constant

msladek commented on component/src/test/java/com/celements/structEditor/OptionTagPageTypeTest.java:
```diff
@@ -0,0 +1,185 @@
+
+  @Test
+  public void test_collectAttributes_none() throws Exception {
+
```
unnecessary empty line

msladek commented on component/src/test/java/com/celements/structEditor/OptionTagPageTypeTest.java:
```diff
@@ -0,0 +1,185 @@
+
+    DocumentReference docRef = new DocumentReference("wikiName", "Celements", "TestXClassName");
+    DocumentReference docRef2 = new DocumentReference("wikiName", "Celements", "gfrt");
+
```
unnecessary empty lines

msladek commented on component/src/test/java/com/celements/structEditor/OptionTagPageTypeTest.java:
```diff
@@ -0,0 +1,185 @@
+  public void test_collectAttributes_field_disabled() throws Exception {
+
+    DocumentReference docRef = new DocumentReference("wikiName", "Celements", "TestXClassName");
+
```
unnecessary empty lines

msladek commented on component/src/test/java/com/celements/structEditor/OptionTagPageTypeTest.java:
```diff
@@ -0,0 +1,185 @@
+        OptionTagEditorClass.FIELD_VALUE))).andReturn(Optional.<String>of("value"));
+    expect(modelAccessMock.getFieldValue(eq(docRef), eq(
+        OptionTagEditorClass.FIELD_LABEL))).andReturn(Optional.<String>of("label"));
+    replayDefault();
```
please be consistent, either add empty lines before replayDefault or don't - but not both mixed in the same class.

I personally prefer an empty line before replayDefault.

msladek commented on component/src/test/java/com/celements/structEditor/SelectTagPageTypeTest.java:
```diff
@@ -0,0 +1,173 @@
+
+  @Before
+  public void prepare_Test() throws Exception {
+
```
unnecessary empty line

msladek commented on component/src/test/java/com/celements/structEditor/SelectTagPageTypeTest.java:
```diff
@@ -0,0 +1,173 @@
+
+  @Test
+  public void test_GetViewTemplateName() {
+    String expectedStr = "SelectTagView";
```
use constant

msladek commented on component/src/test/java/com/celements/structEditor/SelectTagPageTypeTest.java:
```diff
@@ -0,0 +1,173 @@
+
+  @Test
+  public void testGetName() {
+    String expectedStr = "SelectTag";
```
use constant

GalCohen08 committed:
OptionTagPageTest
OptionTagPage
SelectTagPageTest
SelectTagPage

GalCohen08 committed:
StructuredDataEditorScriptServiceTest

GalCohen08 committed:
StructuredDataEditorScriptServiceTest

StructuredDataEditorScriptService

GalCohen08 committed:
DefaultStructuredDataEditorServiceTest

DefaultStructuredDataEditorService

GalCohen08 committed:
DefaultStructuredDataEditorService

GalCohen08 committed:
DefaultStructuredDataEditorServiceTest

GalCohen08 committed:
DefaultStructuredDataEditorServiceTest

test_getCellDateValue_nullValue

GalCohen08 committed:
test_getCellDateValue_nullValue()
test_getCellDateValue_NullValue_ifTitle() 
test_getCellDateValue_NullValue_ifContent()

GalCohen08 committed:
test_getCellDateValue_NotValue

not working yet

GalCohen08 committed:
test_getCellDateValue_NotValue

GalCohen08 committed:
test_resolveFormPrefix_DocumentNotExists
